website/content/docs/configuration/index.mdx

```diff
@@ -187,6 +177,26 @@ to specify where the configuration is.
   participating in a Raft cluster, this header will be omitted, whether this configuration
   option is enabled or not.
 
+- `log_level` `(string: "info")` - Log verbosity level.
+  Supported values (in order of descending detail) are `trace`, `debug`, `info`, `warn`, and `error`.
+  This can also be specified via the `VAULT_LOG_LEVEL` environment variable.
+
+  ~> Note: On SIGHUP (`sudo kill -s HUP` _pid of vault_), if a valid value is specified, Vault will update the existing log level,
+  overriding (even if specified) both the CLI flag and environment variable.
+
+  ~> Note: Not all parts of Vault's logging can have its log level be changed dynamically this way; in particular,
```

Comment:
What's this about?

Reply:
I was just trying to make it more clear about how this works, as previously it wasn't really 'different' in a way that stood out. We don't seem to support reloading lots of parts of Vault's config - so I thought a note would help people not miss it.

Reply:
What did you mean about secrets/auth plugins?

Reply:
It was just prose that was originally there, but I moved it into a note. 

https://developer.hashicorp.com/vault/docs/configuration#log_level

Is that not correct? I can remove it.

Reply:
Ah, I think I get what it's about, namely, external plugins.